homeassistant/components/ring/camera.py

```diff
@@ -131,6 +135,13 @@ async def handle_async_mjpeg_stream(self, request):
 
     async def async_update(self) -> None:
         """Update camera entity and refresh attributes."""
+        if (
+            self._device.has_capability(MOTION_DETECTION_CAPABILITY)
+            and self._attr_motion_detection_enabled != self._device.motion_detection
+        ):
+            self._attr_motion_detection_enabled = self._device.motion_detection
+            self.async_schedule_update_ha_state()
```

Comment:
```suggestion
            self.async_write_ha_state()
```

I think you can write state directly here but thats probably not needed since it will happen at the end of the update anyways